lib/rubocop/cop/lint/circular_argument_reference.rb

```diff
@@ -18,11 +18,13 @@ def on_kwoptarg(node)
           arg_name, arg_value = *node
           case arg_value.type
           when :send
-            # ruby 2.0
-            _, name = *arg_value
-            return unless name == arg_name
+            # Ruby 2.0 will have type send every time, and "send nil" if it is
+            # calling itself with a specified "self" receiver
+            called_class, name = *arg_value
```

Comment:
Guess you missed my previous comment - a better name for this variable would be "receiver".